### DISCUSSION TITLE
Are there alternatives to whisper? I am not good enough results

### DISCUSSION THREAD

**Discussion by NormalFall0** [openai/whisper#2171] [Q&A]
I mentioned my problem for example here: https://github.com/openai/whisper/discussions/2170
What else can I use?

**Comment by glangford** [comment]
AssemblyAI has the Universal-1 model (I have no affiliation with them and haven't tried it)

> Universal-1 is trained on four major languages: English, Spanish, French, and German, and shows extremely strong speech-to-text accuracy in almost all conditions, including heavy background noise, accented speech, natural conversations, and changes in language, while achieving fast turn-around time and improved timestamp accuracy.

https://www.assemblyai.com/blog/announcing-universal-1-speech-recognition-model/
